src/tf.tsx

```diff
@@ -1,15 +1,30 @@
+import { FaceLandmarksDetector } from "@tensorflow-models/face-landmarks-detection";
```

Comment:
`import type { FaceLandmarksDetector } ...`?

Reply:
oh, is that the proper way to do it? i'll update it!